src/qt/forms/receivecoinsdialog.ui

```diff
@@ -114,6 +114,12 @@
              <iconset resource="../bitcoin.qrc">
               <normaloff>:/icons/receiving_addresses</normaloff>:/icons/receiving_addresses</iconset>
             </property>
+            <property name="autoDefault">
+             <bool>false</bool>
+            </property>
```

Comment:
I don't think we need to mess with this?

Reply:
This change _is_ required.

From Qt [docs](https://doc.qt.io/qt-5/qpushbutton.html#autoDefault-prop):
> **autoDefault : bool**
> ...
> This property's default is true for buttons that have a QDialog parent; otherwise it defaults to false.

 As `ReceiveCoinsDialog` is derived from `QDialog` its buttons have `autoDefault == true` by default.

Having `autoDefault == true` breaks rendering of the default button when tabbing through it (when it is focused out). At least on Linux Mint 20 with system Qt 5.12.8.

Reply:
Weird, sounds like a Qt bug?

Reply:
Maybe. Didn't track it down though.